packages/asciidoc.rb

```diff
@@ -3,32 +3,34 @@
 class Asciidoc < Package
   description 'AsciiDoc is a presentable text document format for writing articles, UNIX man pages and other small to medium sized documents.'
   homepage 'http://asciidoc.org/'
-  version '8.6.9-1'
+  @_ver = '9.0.5'
+  version @_ver
   compatibility 'all'
-  source_url 'https://downloads.sourceforge.net/project/asciidoc/asciidoc/8.6.9/asciidoc-8.6.9.tar.gz'
-  source_sha256 '78db9d0567c8ab6570a6eff7ffdf84eadd91f2dfc0a92a2d0105d323cab4e1f0'
+  source_url "https://github.com/asciidoc/asciidoc-py3/releases/download/#{@_ver}/asciidoc-#{@_ver}.tar.gz"
+  source_sha256 '1a20647eb62ca37bc8107affab968caa0f674f0e962b497d1674391f636c7038'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-8.6.9-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-8.6.9-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-8.6.9-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-8.6.9-chromeos-x86_64.tar.xz',
+     aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-9.0.5-chromeos-armv7l.tar.xz',
+      armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-9.0.5-chromeos-armv7l.tar.xz',
+        i686: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-9.0.5-chromeos-i686.tar.xz',
+      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/asciidoc-9.0.5-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: '913147e51102cd8938ac8c3dcf7520c2995c081b1ba9ee097539cf83b3557a96',
-     armv7l: '913147e51102cd8938ac8c3dcf7520c2995c081b1ba9ee097539cf83b3557a96',
-       i686: '24df5bebe051fe916d7bf722b85bea5e0b8dbe099e2b95e2fbee609b22c14639',
-     x86_64: '048de56fd444c9c5542270fb8080b23f0264467e3c10853b9925b4add914b053',
+     aarch64: '60bba587c331dc462a70dd59a6b7cc68cc08c79a836dde47fe07de6735daa090',
+      armv7l: '60bba587c331dc462a70dd59a6b7cc68cc08c79a836dde47fe07de6735daa090',
+        i686: '48be337c603382c032d4aaebd0ff4e47341a82aa7429f518c83e1860b390d276',
+      x86_64: 'a9fabec30cf8243847e065188568f048d3323f2cc52e47cc8a8aa516b68957cb',
   })
 
   depends_on 'autoconf' => :build
   depends_on 'compressdoc' => :build
-  depends_on 'python27' unless File.exists? "#{CREW_PREFIX}/bin/python"
+  depends_on 'python3'
```

Comment:
None of these dependencies need to be listed here.

Reply:
I'll get rid of these. I didn't clean out the preexisting depends for the most part.